lib/diagnostic.rb

```diff
@@ -45,7 +54,12 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+
+class Developer < Person
+    def solve_problems
+      puts 'think think think'
```

Comment:
This will return `nil` since `puts` doesn't have a return value.  Since we want to return a string, we can explicitly return it with `return 'think think think'` or implicitly return it with `'think think think'`.